docs/source/getting_started/content/con_understanding-codeready-containers-configuration.adoc

```diff
@@ -0,0 +1,10 @@
+[id="understanding-codeready-containers-configuration_{context}"]
+= Understanding {prod} configuration
+
+Both the [command]`{bin}` binary and the {prod} virtual machine can be configured using the [command]`{bin} config` command.
```

Comment:
Passive voice "can be configured". Suggestion:

You can use the [command]`{bin} config` command to configure both the [command]`{bin}` binary and the {prod} virtual machine.